### ISSUE TITLE
Add syntax to enable image resizing

### ISSUE THREAD

**Issue by flexd** [russross/blackfriday#265]
Something like `![some image](http://google.com/someurl 500x400)` resulting in HTML `<img src="http://google.com/someurl" width=500 height=200/>` would be great to have. I know it's better to resize images before you publish, but it would be really handy to have.

**Comment by shurcooL** [comment]
@flexd asked me on Gophers Slack about this issue:

> hey, saw you had looked at the blackfriday code a bit, do you think this would be hard to implement?

I think the implementation difficulty is medium.

However, in order to implement this, this requires changing the Markdown spec. I'm not sure what to say about that.

**Comment by flexd** [comment]
Yeah I figured HTML is the easiest way of doing so. I know that a few other markdown processors implement a syntax similar to what I suggested. I guess it's up to the project if they want to include some functionality or stick to the spec.
